src/ManageCourses.Api/Startup.cs

```diff
@@ -191,7 +191,8 @@ private void Migrate(ManageCoursesDbContext dbContext)
                     {
                         delayMs = maxDelayMs;
                     }
-                    _logger.LogError($"Failed to apply EF migrations. Attempt {migrationAttempt} of ∞. Waiting for {delayMs}ms before trying again.", ex);
+                    // exception included in message string because app insights isn't showing the messages and kudo log stream only shows the message string.
```

Comment:
I think you need something like this

Reply:
https://github.com/DFE-Digital/manage-courses-ui/blob/a1b03b3073e78f4d51fab86ca82ee798758093ab/src/ui/ActionFilters/McExceptionFilter.cs#L47

Reply:
You swallowed the exception, nothing bubbles up...

Makes sense that its not on show

Reply:
nice one

Reply:
done